docs/source/getting-started.md

```diff
@@ -76,23 +76,39 @@ conda uninstall jupyter_enterprise_gateway
+To leverage the full distributed capabilities of Spark, Jupyter Enterprise Gateway has provided deep integration with various resource managers. Having said that, Enterprise Gateway also supports running in a pseudo-distributed mode utilizing for example both YARN client or Spark Standalone modes. We've also recently added Kubernetes, Docker Swarm and IBM Spectrum Conductor integrations.
 
-* [Installing and Configuring kernels](getting-started-kernels.html)
+Please follow the links below to learn specific details about how to enable/configure the different modes of distributing your kernels:
 
-### Configuring Resource Managers
+* [Vanilla](kernel-vanilla.html)
+* [YARN Cluster Mode](kernel-yarn-cluster-mode.html)
+* [YARN Client Mode](kernel-yarn-client-mode.html)
+* [Standalone](kernel-spark-standalone.html)
+* [Kubernetes](kernel-kubernetes.html)
+* [Docker Swarm](kernel-docker.html)
+* [IBM Spectrum Conducto](kernel-conductor.html)
 
-To leverage the full distributed capabilities of Spark, Jupyter Enterprise Gateway has provided deep integration with the Apache Hadoop YARN resource manager. Having said that, Enterprise Gateway also supports running in a pseudo-distributed mode utilizing both YARN client or Spark Standalone modes. We've also recently added Kubernetes, Docker Swarm and IBM Spectrum Conductor integrations.
+In each of the resource manager sections, we set the `KERNELS_FOLDER` to `/usr/local/share/jupyter/kernels` since that's one of the default locations searched by the Jupyter framework.  Co-locating kernelspecs hierarchies in the same parent folder is recommended, although not required.
 
-Please follow the links below to learn specific details about how to enable/configure the different modes of distributing your kernels:
+**Important requirements regarding the nodes**
+
+We have two cases:
 
-* [Enabling YARN Cluster Mode support](kernel-yarn-cluster-mode.html)
-* [Enabling YARN Client Mode or Spark Standalone support](kernel-yarn-client-mode.html)
-* [Enabling Kubernetes Support](kernel-kubernetes.html)
-* [Enabling Docker Swarm Support](kernel-docker.html)
-* [Enabling IBM Spectrum Conductor Support](kernel-conductor.html)
+1. The kernel is run in (Docker) container: in that case, the image should ensure the availability of the kernel libraries and kernelspec. There is no additional action to take on the node.
+2. The kernel is not run in a (Docker) container, e.g. Vanilla across a set of hosts, YARN (cluster mode, client mode) or Spark Standalone: All kernels (libraries...) and their corresponding kernelspecs must reside on each nodes. This requirement is applicable for vanilla using a set of distributed hosts and for all reource managers. The kernelspec hierarchies (i.e., paths) must be available and identical on all nodes. This is not applicable to Apache Toree in Yarn cluster mode since spark-submit will transfer the files at startup.
```

Comment:
The kernelspecs are only required on all nodes if using the DistributedProcessProxy - which can apply to YARN Client mode, Standalone, and _Vanilla_.  (See comment at start of Vanilla section)